modules/View.js

```diff
@@ -137,16 +138,87 @@
      * @method prepare
      */
     prepare: function() {
-      if (this.model) {
-        return {
-          model: this.model.toJSON(),
-          view: this.viewState.toJSON()
-        };
-      } else {
-        return {
-          view: this.viewState.toJSON()
-        };
+      return this.__getPrepareFieldsContext();
+    },
+
+    /**
+     * Parses the combined arrays from the defaultPrepareFields array and the prepareFields array (or function
+     * returning an array).
+     *
+     * The default prepared fields are: [ { name: 'view', value: 'viewState' }, 'model' ]
+     *
+     * Prepared fields can be defined in a couple of ways:
+     *   preparedFields = [
+     *     'model',
+     *     { name: 'app', value: someGlobalCell },
+     *     'a value that does not exist on the view',
+     *     { name: 'view', value: 'viewState' },
+     *     { name: 'patientId', value: '_patientId' },
+     *     'readOnly'
+     *   ]
+     *
+     * Will result in the following context (where this === this view and it assumes all the properties on the view
+     * that are referenced are defined):
+     *   {
+     *     model: this.model.toJSON(),
+     *     app: someGlobalCell.toJSON(),
+     *     view: this.viewState.toJSON(),
+     *     patientId: this._patientId,
+     *     readOnly: this.readOnly
+     *   }
+     *
+     * Things to be careful of:
+     *   * If someGlobalCell is a property on the view than the property on the view will be used instead.
```

Comment:
then not than. Also, add quotes around it. If the view already has a field named 'someGlobalCell'...

Reply:
done